client/blocks/inline-help/style.scss

```
@@ -15,7 +15,7 @@
 		border-radius: 100%;
 		background: var( --color-primary );
 		box-shadow: 0 2px 6px rgba( 0, 0, 0, 0.15 );
-		border: 1px solid darken( $blue-wordpress, 5 );
+		border: 1px solid var( --color-primary-500 );
```

```suggestion
		border: 1px solid var( --color-primary );
```
I think this is equivalent to the 500 value, so the 500 is not needed